package.json

```diff
@@ -10,21 +10,24 @@
   },
   "devDependencies": {
     "@popperjs/core": "^2.11.6",
-    "bootstrap": "^5.2.3",
+    "bootstrap": "^5.3.1",
     "husky": "^8.0.3",
     "nuxt": "^3.6.5",
-    "sass": "^1.58.3",
+    "sass": "^1.64.2",
     "sass-loader": "^13.2.0"
   },
   "dependencies": {
     "@nuxt/kit": "^3.6.5",
     "@phosphor-icons/vue": "^2.1.4",
     "@pinia/nuxt": "^0.4.11",
     "@types/node": "^14.18.0",
+    "bootstrap-icons": "^1.10.5",
```

Comment:
We don't use bootstrap-icons but https://phosphoricons.com/

Reply:
Removed the package, added in .vue file
`import { PhUserCircle, PhLock, PhEye, PhEyeSlash } from "@phosphor-icons/vue";`